extensions/arc/deployment/src/test/java/io/quarkus/arc/test/lookup/ListInjectionTest.java

```diff
@@ -23,6 +23,7 @@
 import io.quarkus.arc.Arc;
 import io.quarkus.arc.InstanceHandle;
 import io.quarkus.arc.Priority;
+import io.quarkus.arc.Unremovable;
 import io.quarkus.test.QuarkusUnitTest;
 
 public class ListInjectionTest {
```

Comment:
I think that we should also extend the `io.quarkus.arc.test.all.ListAllTest` from the `arc-tests` module.

Reply:
In what manner exactly? Basically keeping them 1:1 in what they test?

Reply:
Yes, just add the `@Injec @All` variant there...

Reply:
Done, added a test.